packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/white-label-manager/white-label-manager.html

```diff
@@ -0,0 +1,45 @@
+<section>
+    <oui-header
+        data-heading="{{ :: 'telephony_white_label_manager_title' | translate }}"
+        data-description="{{ :: 'telephony_white_label_manager_descrition' | translate }}"
+    >
+        <oui-back-button
+            data-href="{{ OptionsGroupCtrl.$state.href('telecom.telephony.billingAccount.administration') }}"
```

Comment:
**nitpick:** Maintain the URL in the `$ctrl` associated with this template.

Reply:
I fixed.